.github/workflows/cs.yml

```diff
@@ -0,0 +1,41 @@
+name: CS
```

Comment:
```suggestion
name: Coding Standards
```

Reply:
Is there any reason to not be more explicit here? Since this is displayed in the UI, it may help to write out the full name.

Reply:
The reason - for me - was that the name in the button on the Readme then becomes very very long, but I'm not fussed either way.